lib/release_tasks.ex

```diff
@@ -0,0 +1,70 @@
+defmodule GoldenPath.ReleaseTasks do
```

Comment:
This is an example task within a release that can be used remotely. Since we can't use "mix bah", we could ssh into a remote shell and run a migrate command.

Gigalixir provides a proxy command for us via ```$gigalixir ps:migrate```, but since this is probably one of the more common needed commands, it's provided as an example.